gradle/libs.versions.toml

```diff
@@ -2,14 +2,15 @@
 autoService = "1.0"
 gjf = "1.11.0"
 jvmTarget = "1.8"
-kotlin = "1.6.0"
+kotlin = "1.6.10"
+dokka = "1.6.10"
```

Comment:
This isn't reused anywhere, please just keep the version in the dependency declaration like before

Reply:
Ok, deleted the `dokka` variable. Updating the kotlin version solved the lockstep issue. The intention is to keep the kotlin and dokka version same.

Reply:
Sure, but like I said - they are not updated at the same time